WorkloadManagementSystem/JobWrapper/Watchdog.py

```diff
@@ -363,87 +369,85 @@ def _performChecks( self ):
-    staticParamDict = {'StandardOutput':recentStdOut}
-    self.__sendSignOfLife( int( jobID ), heartBeatDict, staticParamDict )
-    return S_OK( 'Watchdog checking cycle complete' )
+    staticParamDict = {'StandardOutput': recentStdOut}
+    self.__sendSignOfLife(int(jobID), heartBeatDict, staticParamDict)
+    return S_OK('Watchdog checking cycle complete')
 
   #############################################################################
-  def __getCPU( self ):
+  def __getCPU(self):
     """Uses os.times() to get CPU time and returns HH:MM:SS after conversion.
     """
     try:
-      cpuTime = self.processMonitor.getCPUConsumed( self.wrapperPID )
+      cpuTime = self.processMonitor.getCPUConsumed(self.wrapperPID)
       if not cpuTime['OK']:
-        self.log.warn( 'Problem while checking consumed CPU' )
+        self.log.warn('Problem while checking consumed CPU')
         return cpuTime
       cpuTime = cpuTime['Value']
       if cpuTime:
-        self.log.verbose( "Raw CPU time consumed (s) = %s" % ( cpuTime ) )
-        return self.__getCPUHMS( cpuTime )
+        self.log.verbose("Raw CPU time consumed (s) = %s" % (cpuTime))
+        return self.__getCPUHMS(cpuTime)
       else:
-        self.log.error( "CPU time consumed found to be 0" )
+        self.log.error("CPU time consumed found to be 0")
         return S_ERROR()
 
     except Exception as e:
-      self.log.warn( 'Could not determine CPU time consumed with exception' )
-      self.log.exception( e )
-      return S_ERROR( "Could not determine CPU time consumed with exception" )
-
-
+      self.log.warn('Could not determine CPU time consumed with exception')
+      self.log.exception(e)
+      return S_ERROR("Could not determine CPU time consumed with exception")
 
   #############################################################################
-  def __getCPUHMS( self, cpuTime ):
-    mins, secs = divmod( cpuTime, 60 )
-    hours, mins = divmod( mins, 60 )
-    humanTime = '%02d:%02d:%02d' % ( hours, mins, secs )
-    self.log.verbose( 'Human readable CPU time is: %s' % humanTime )
-    return S_OK( humanTime )
+  def __getCPUHMS(self, cpuTime):
+    mins, secs = divmod(cpuTime, 60)
+    hours, mins = divmod(mins, 60)
+    humanTime = '%02d:%02d:%02d' % (hours, mins, secs)
+    self.log.verbose('Human readable CPU time is: %s' % humanTime)
+    return S_OK(humanTime)
 
   #############################################################################
-  def __interpretControlSignal( self, signalDict ):
+  def __interpretControlSignal(self, signalDict):
     """This method is called whenever a signal is sent via the result of
        sending a sign of life.
     """
-    self.log.info( 'Received control signal' )
-    if type( signalDict ) == type( {} ):
-      if signalDict.has_key( 'Kill' ):
-        self.log.info( 'Received Kill signal, stopping job via control signal' )
+    self.log.info('Received control signal')
+    if isinstance(signalDict, type({})):
```

Comment:
I know you didn't code this line, but can you please change it in 
```
if isinstance(signalDict, dict):
```
?

As it is it looks funny